src/providers/virtualraster/qgsvirtualrasterprovider.h

```diff
@@ -3,7 +3,7 @@
      --------------------------------------
     Date                 : June 10, 2021
     Copyright            : (C) 2021 by Francesco Bursi
-    email                : francesco.bursi
+    email                : ffrancesco dot bursi at hotmail dot it
```

Comment:
another typo

Reply:
it seems that I'm not able to write my name... sorry for that.